### PR TITLE
faker implemented for names & zips

### PR DESCRIPTION
**Pull request by jacobmovingfwd**
Generates random names, and random TX Zips.
I haven't found a good db to align Zips to Cities, but I doubt any data checking would validate that.

### CODE REVIEW

**Comment by ramblingjordan** [comment]
@jacobmovingfwd sorry to do this! I thought this was able to be merged but then I approved another PR and I think it broke this. Can you check on the merge conflicts?

File: data.py [hunk 1]
```diff
@@ -1,209 +1,62 @@
...
-
-def sign_up_page():
-  raise NotImplementedError()
+#!/usr/bin/env python3
+import random
+import time
+import json
+from faker import Faker
+
+fake = Faker('en-US')
```
Comment (coolgeek6667) [inline data.py:7]: Should add locale `es_MX` to improve the pool of names

**Comment by santaanna2021** [comment]
Zip code listings for all non PO Box zips 

[TX_Zips.csv](https://github.com/SeanDaBlack/AbBOT/files/7108656/TX_Zips.csv)

**Comment by ramblingjordan** [comment]
We have moved active development of the python bot to this repo: https://github.com/ramblingjordan/AbBOT-python

Could you please recreate your PR over there?

Thanks!

**Pull request closed without merging**